src/naive/datetime/mod.rs

```diff
@@ -328,6 +328,16 @@ impl NaiveDateTime {
         parsed.to_naive_datetime_with_offset(0) // no offset adjustment
     }
 
+    /// Parses a string with the specified format string and returns a new `NaiveDateTime`, and a
+    /// slice with the remaining portion of the string.
+    /// See the [`format::strftime` module](../format/strftime/index.html)
+    /// on the supported escape sequences.
```

Comment:
I noticed in some cases the original `fn parse_from_str` has a larger and longer docstring with further details. Could you hint to the user that `fn parse_and_remainder` is very similar to `fn parse_from_str` in the docstring?

Could you add a separate line like:

```lang-rust
///
/// Similar to [`parse_from_str`](parse_from_str)
```

Do this for all the `fn parse_from_str` docstrings.